APIs/schemas/token_schema.json

```diff
@@ -0,0 +1,58 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "JSON Web Token Contents",
+  "description": "Claims contained within JSON Web Token",
+  "type": "object",
+  "properties": {
+    "iss": {
+      "description": "The DNS name of the Authorization Server issuing the token",
+      "type": "string",
+      "format": "uri"
+    },
+    "sub": {
+      "description": "The unique identifier assigned to the end-user by the user authorization system",
+      "type": "string"
+    },
+    "aud": {
+      "description": "The fully resolved domain name of the intended recipient, or a domain name containing wild-card characters",
+      "type": "string"
+    },
+    "exp": {
+      "description": "The UTC epoch time at which the token expires",
+      "type": "number"
+    },
+    "iat": {
+      "description": "The UTC epoch time at which the token was issued",
+      "type": "number"
+    },
+    "client_id": {
+      "description": "The client identifier of the OAuth 2.0 client that requested the token",
+      "type": "string"
+    },
+    "scope": {
+      "description": "A string containing a space-separated list of scopes associated with the token",
+      "type": "string"
+    },
+    "x-nmos-api": {
+      "description": "An object containing the access permissions of the user for the listed NMOS APIs",
+      "type": "object",
+      "patternProperties": {
+      	"[a-zA-Z]": {
```

Comment:
Could you fix the indentation here? I wonder if we can also get away with disallowing uppercase? It's not used in any APIs to date to my knowledge, and certainly not in the key that would go here.

Reply:
DID_SDID (but I still agree)

Reply:
Ha, good point

Reply:
Hmm, though we have `-` in `x-nmos-api` and `_` elsewhere, so maybe one or both of those need including?